examples/plot_connectivity.py

```diff
@@ -166,3 +166,52 @@
 #
 
 #   <a href="https://github.com/neuropycon/graphpype" target="_blank">graphpype</a>
+
+##############################################################################
+from ephypype.gather.gather_results import get_results  # noqa
+from ephypype.gather.gather_results import get_channel_files  # noqa
+from ephypype.aux_tools import parse_string  # noqa
+from visbrain.objects import ConnectObj, SourceObj, SceneObj, ColorbarObj  # noqa
+
+th = .8
```

Comment:
I would add some text to explain what this threshold is why we need it. Also maybe it's better to call it `thresh` instead of `th`?

Reply:
@EtienneCmb the threshold is related to the number of edges to show. Is it?